### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by katrinamariehh**
Made updates to the documentation to clarify instructions around deployment; a remaining detail is that I was able to get it all to work by getting a `.env` file from Sean/Chris, that maybe needs to be stored in a more reliable way.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,13 +3,43 @@
...
+
+First, you'll need some tools to get the system built:
+
+- Docker: [Docker for Mac][docker] is nice; otherwise install both `docker` and
+  `docker-compose` with your favorite package manager
+- VALID AWS credentials with write permissions to both S3 and SQS
+- [AWS CLI tool][awscli] (also available in Homebrew)
+
+In addition to cloning this repository, you'll need to clone
+the [data-processor repository][data-processor] and
```
Comment (sduckett) [inline README.md]: I believe you're looking for `sms-web` and `sms-worker`, rather than `data-processor` and `metis`.

**Review by sduckett — CHANGES_REQUESTED** [review]
If this pointed to the right repos, I'd 👍

File: README.md (hunk 1 above)
Comment (katrinamariehh) [reply README.md]: Good call!  That's what I get for copy-pasting.

File: README.md [hunk 2]
```diff
@@ -146,7 +146,8 @@ environment variables so that your Docker client communicates with the
 swarm: `DOCKER_HOST`, `DOCKER_CERT_PATH`, and `DOCKER_TLS_VERIFY`.
 
 * `DOCKER_HOST` should include the protocol `tcp`, and the
-port. (e.g.: `tcp://123.456.789.101:3376`)
+port. (e.g.: `tcp://123.456.789.101:3376`); this will be the IP address of
+the ec2 instance that is the swarm master.
```
Comment (sduckett) [inline README.md:150]: yeah, good call!

**Review by sduckett — APPROVED** [review]

**Pull request merged**